modules/seqexec/web/client/src/main/resources/less/style.less

```diff
@@ -383,52 +372,52 @@ body {
   .borderTop();
   .borderRight();
   background-color: @white;
```

Comment:
I'm curious why you use `@white` here (which I assume is being substituted in) but `lightcyan` (hard-coded?) on L364.

Reply:
`@white` is a less variable and it is relevant if we ever support theming. A dark theme may define `@white` as `black` though that may sound strange.

There is no variable for `lightcyan`, so that woudn't be themable

Reply:
Theming would be great, but wouldn't it be detrimental, then, to use any colours that are not less variables and thus not themable?

Reply:
it would, theming would have two parts, theming at the semantic-ui level where those variables are used. We should also need to theme our own css and there we may need to replace `lightcyan` for a variable too